services/director/tests/test_registry_proxy.py

```diff
@@ -108,9 +106,9 @@ def test_get_service_last_namess():
     repo = "services/dynamic/modeler"
     assert registry_proxy.get_service_last_names(repo) == "invalid service"
 
-@pytest.mark.asyncio
+
 async def test_get_service_details(push_services, configure_registry_access):
-    images = push_services(1, 1)    

```

Comment:
so here if I write now:
```python
import pdb; pdb.set_trace()
```
the debugger does not stop anymore and the test passes (under windows using conda/ and unser windows under linux subsystem). I'm probalby missing something there.

Reply:
will try tomorrow in my linux computer ... definitively scary!